h/auth/__init__.py

```diff
@@ -41,6 +44,15 @@ def includeme(config):
     global DEFAULT_POLICY
     global WEBSOCKET_POLICY
 
+    config.register_service_factory('.services.auth_ticket_service_factory',
+                                    iface=pyramid_authsanity.interfaces.IAuthService)
+
+    # Set up authsanity
+    settings = config.registry.settings
+    settings['authsanity.source'] = 'cookie'
```

Comment:
We probably also want to set `authsanity.cookie.max_age` and `authsanity.cookie.httponly` in order preserve behaviour with the current implementation.